src/tests/auth.test.ts

```diff
@@ -23,11 +23,6 @@ beforeEach(async () => {
 afterEach(async () => {
   setTimeout(() => {}, 500);
```

Comment:
might as well remove this as well if we're using mockImplementation

Reply:
gotcha